src/main/webapp/WEB-INF/cron.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<cronentries>
+  <cron>
+    <url>/cron/email</url>
+    <description>Send email with resume of the day for each technology followers</description>
+    <schedule>every 1 minutes</schedule>
```

Comment:
@tsr-thulio schedule to some hour in a day. Every minute will raise requests in our app and CI&T will have to pay more.